src/componenets/LinkAdd.tsx

```diff
@@ -2,14 +2,18 @@ import "./LinkAdd.css";
 import link from "../images/link.svg";
 import { useState } from "react";
 
-function LinkAdd({ toggleModal }) {
+interface ToggleModal {
+  toggleModal: (id: number) => void;
+}
+
+function LinkAdd({ toggleModal }: ToggleModal) {
   const [inputValue, setInputInput] = useState("");
 
-  function handleInputChange(event) {
+  function handleInputChange(event: React.ChangeEvent<HTMLInputElement>) {
```

Comment:
보통은 React를 제거하고 이렇게 표현해주는 편입니다~
React를 사용해도 되지만, 그러면 useState, useEffect도 React.useState, React.useEffect와 같이 일관되게 작성하는게 맞다고 생각해요!
```suggestion
  function handleInputChange(event: ChangeEvent<HTMLInputElement>) {
```